features/api.feature

```diff
@@ -7,12 +7,12 @@ Feature: Client behaviour for the API endpoints
     When I verify that running `pro api invalid.endpoint` `with sudo` exits `1`
     Then stdout matches regexp:
     """
-    {\"_schema_version\": \"v1\", \"data\": {}, \"errors\": \[{\"code\": \"api\-invalid\-endpoint", \"meta\": {}, \"title\": \"'invalid\.endpoint' is not a valid endpoint\"}], \"result\": \"failure\", \"version\": \".*\", \"warnings\": \[]}
```

Comment:
Can we add an integration test where we setup an env var ?

Reply:
sure

Reply:
done